core/includes/form.inc

```diff
@@ -128,7 +128,36 @@ function backdrop_get_form($form_id) {
   array_shift($args);
   $form_state['build_info']['args'] = $args;
 
-  return backdrop_build_form($form_id, $form_state);
+  // Disable redirection if the request was made via AJAX.
+  $ajax_post = isset($_POST['ajax_page_state']) && $_POST['form_id'] === $form_id;
```

Comment:
Okay, `$_POST['form_id']` needs an isset or empty check.
When managing blocks on a layout, if you click on any "Configure" link or click on any item after clicking any "Add block" link, a watchdog notice similar to this is generated:
`Notice: Undefined index: form_id in backdrop_get_form() (line 132 of /home/qa/github/backdrop/backdrop/pr/2918/core/includes/form.inc).`
The form appears to work as expected otherwise.

Reply:
I found a place where this error does affect the behavior of Backdrop:
1. Navigate to the Install modules page at /admin/modules/install
2. Click the name of a module
3. Click the Add to Installation queue link

The module does not get added to the queue.